### PR TITLE
Migrate to Swift 2.3

### PR DESCRIPTION
**Pull request by jessesquires**
🎉

### CODE REVIEW

**Comment by jessesquires** [comment]
Hopefully travis succeeds — I'm not sure if the iOS 8 and 9 simulators are installed for Xcode8

File: Source/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.0.1</string>
+	<string>5..0.0</string>
```
Comment (dcaunt) [inline Source/Info.plist]: Typo here (extra `.` after 5)

Comment (jessesquires) [reply Source/Info.plist]: ugh. thanks @dcaunt ! 🙌

**Comment by jessesquires** [comment]
man this is taking forever

**Comment by dcaunt** [comment]
Good old Travis

**Comment by jessesquires** [comment]
ah! we're good on simulators:

https://docs.travis-ci.com/user/osx-ci-environment/#Xcode-8

🎉 

now we just need to wait forever

**Comment by dcaunt** [comment]
This may speed up if we cancel the builds for the commit penultimate commit.

**Comment by jessesquires** [comment]
Got most passing here:

https://travis-ci.org/jessesquires/JSQDataSourcesKit/builds/159637483

First failure was podspec validation (will get fixed once we tag).

Some of the other builds have timed out. Going to merge and cancel except for most recent commit.

**Pull request merged**